lib/puppet/network/http/api/v3.rb

```diff
@@ -14,9 +14,13 @@ class Puppet::Network::HTTP::API::V3
     Environments.new(Puppet.lookup(:environments))
   end)
 
-  def self.routes
-    Puppet::Network::HTTP::Route.path(%r{/v3}).
+  def self.master_routes
+    Puppet::Network::HTTP::Route.path(%r{v3}).
         any.
         chain(ENVIRONMENTS, INDIRECTED)
   end
+
+  def self.ca_routes
+    Puppet::Network::HTTP::Route.path(%r{v1}).any.chain(INDIRECTED)
```

Comment:
I'm wondering if it's going to be problematic at all to have these coupled together in a file named after the version of the master's (but not the CA's) API.

Maybe I'm overthinking it but what happens if we need to bump CA to v2?  Would we make the change in this file and leave it as `http/api/v3`?